package.json

```diff
@@ -13,11 +13,9 @@
     "axios": "^0.19.2",
     "dotenv": "^8.2.0",
     "express": "^4.17.1",
-    "mongoose": "^5.9.9",
-    "sequelize": "^5.21.6",
-    "sqlite3": "^4.1.1"
+    "mongoose": "^5.9.9"
   },
   "devDependencies": {
     "nodemon": "^2.0.3"
   }
-}
+}
```

Comment:
I liked that new line at the end. It helps reading files with `cat`

Reply:
okay I am on it